Source/JavaScriptCore/wasm/js/JSWebAssemblyArray.h

```diff
@@ -70,12 +85,23 @@ class JSWebAssemblyArray final : public JSNonFinalObject {
 
     DECLARE_VISIT_CHILDREN;
 
-    Wasm::Type elementType() const { return m_elementType; }
+    Wasm::FieldType elementType() const { return m_elementType; }
     size_t size() const { return m_size; }
 
-    EncodedJSValue get(uint32_t index)
-    {
-        switch (m_elementType.kind) {
+    const FixedVector<uint8_t>& payload() const { return m_payload8; }
```

Comment:
I think this is likely a rebase artifact and can be deleted; the `payload()` method was deleted in merged array patches.